### PR TITLE
Prevent everything from breaking if one feature breaks

### LINKED ISSUE #678: Make features more resilient to failure
In https://github.com/sindresorhus/refined-github/issues/675, one tiny feature broke and took down all of RG. Would be nice if we could somehow compartmentalize each feature, so if one feature broke, only that feature would be broken.

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #678 

I used the `Promise` instead of a plain `try{ fn() } catch(e){ console.error(e) }` because the promise won't actually catch the error so they're easier to debug. Happy to discuss this.

~~The downside is that functions are called asyncronously.~~ Not anymore with `new Promise`

### CODE REVIEW

**Comment by bfred-it** [comment]
Visual explanation of 14066fd:

> 
new Promise is sync; .then is async

File: src/libs/utils.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ import select from 'select-dom';
 import elementReady from 'element-ready';
 import domLoaded from 'dom-loaded';
 
+/**
+ * Prevent fn's errors from blocking the remaining tasks.
+ * https://github.com/sindresorhus/refined-github/issues/678
+ * The code looks weird but it's synchronous and fn is called without args.
+ */
+export const safely = fn => new Promise(resolve => resolve(fn()));
```
Comment (sindresorhus) [inline src/libs/utils.js]:
```js
export const safely = async fn => fn();
```

screen shot 2017-10-17 at 13 42 18

Comment (sindresorhus) [reply src/libs/utils.js]: This would have been much nicer with decorators:

```js
@safely
addYoursMenuItem();
```

**Comment by sindresorhus** [comment]
Yay!

File: src/libs/utils.js (hunk 1 above)
Comment (bfred-it) [reply src/libs/utils.js]: We have no decorators but this is also an option 😋 

```js
async () =>
addYoursMenuItem();
```

**Pull request merged**